### DISCUSSION TITLE
🔗 🚀 Feature Request: Taxonomy Filter doesn't work for the Current Query Source for Loop Grid (Duplicate of #23418)

### DISCUSSION THREAD

**Discussion by rajeshkumarbasani** [elementor/elementor#23707] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Dear team,

When we use taxonomy filter for product grid doesn't filter 

### Describe the solution you'd like

Taxonomy filter has to be work for product loop grid with current query source and also had new filter price range, category, tags, featured/new

thank you

**Comment by OdYnXien** [comment]
I had this same problem. Can't exclude terms when I have current query selected.

**Comment by nicholaszein** [comment]
Hello there, @goklassifieds, and @OdYnXien!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23418

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
